introduction.md

```diff
@@ -1 +1,3 @@
 My favorite Code School path is JavaScript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?